src/App.tsx

```diff
@@ -21,6 +45,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={(event) => {
+              setPerPage(+event.target.value);
```

Comment:
use explicit type conversion
```suggestion
              setPerPage(Number(event.target.value));
```